src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -69,7 +69,7 @@ private AssertionMessages() {
   private static final String WDS_FOOTER_HEADER_CLASS = "wds-global-footer__header";
   private static final By MERCURY_SKIN = By.cssSelector(".mobile-wiki");
   private static final String LOGGED_IN_USER_SELECTOR_MERCURY =
-      ".wds-global-navigation__modal-control-user .wds-avatar img[title=%userName%], .wds-global-navigation__modal-control-user .wds-avatar__inner-border[title=%userName%]";
```

Comment:
This one is pretty significant, since logging tests are checking for logged in/logged out states by checking userName from the avatar. Please check if logging in/logging out works fine with this change. If it's okay then LGTM :)